src/base/QXmppMessage.h

```diff
@@ -243,6 +245,10 @@ class QXMPP_EXPORT QXmppMessage : public QXmppStanza
     QString spoilerHint() const;
     void setSpoilerHint(const QString &);
 
+    // XEP-0407: Mediated Information eXchange (MIX): Miscellaneous Capabilities
+    std::optional<QXmppMixInvitation> mixInvitation() const;
+    void setMixInvitation(const QXmppMixInvitation &mixInvitation);
```

Comment:
This should also be an `std::optional`, so you can remove the invitation with `std::nullopt`.